icons/threejs/threejs-original.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="128pt" height="128pt" version="1.1" viewBox="0 0 128 128"><g id="surface1"><path style="fill:none;stroke-width:4;stroke-linecap:butt;stroke-linejoin:round;stroke:#000;stroke-opacity:1;stroke-miterlimit:10" d="M 63.022741 200.607088 L 19.804399 25.671417 L 193.037634 75.547253 Z M 63.022741 200.607088" transform="matrix(0.564449,0,0,0.564449,5.059717,2.40043)"/><path style="fill:none;stroke-width:4;stroke-linecap:butt;stroke-linejoin:round;stroke:#000;stroke-opacity:1;stroke-miterlimit:10" d="M 106.393334 50.612796 L 127.978284 138.108313 L 41.41703 113.160014 Z M 106.393334 50.612796" transform="matrix(0.564449,0,0,0.564449,5.059717,2.40043)"/><path style="fill:none;stroke-width:4;stroke-linecap:butt;stroke-linejoin:round;stroke:#000;stroke-opacity:1;stroke-miterlimit:10" d="M 84.912192 125.028622 L 74.185461 81.568063 L 117.19619 93.914184 Z M 84.912192 125.028622" transform="matrix(0.564449,0,0,0.564449,5.059717,2.40043)"/><path style="fill:none;stroke-width:4;stroke-linecap:butt;stroke-linejoin:round;stroke:#000;stroke-opacity:1;stroke-miterlimit:10" d="M 63.458731 38.155947 L 74.178541 81.616507 L 31.174733 69.270386 Z M 63.458731 38.155947" transform="matrix(0.564449,0,0,0.564449,5.059717,2.40043)"/><path style="fill:none;stroke-width:4;stroke-linecap:butt;stroke-linejoin:round;stroke:#000;stroke-opacity:1;stroke-miterlimit:10" d="M 149.473267 62.931234 L 160.193077 106.391793 L 117.189269 94.045673 Z M 149.473267 62.931234" transform="matrix(0.564449,0,0,0.564449,5.059717,2.40043)"/><path style="fill:none;stroke-width:4;stroke-linecap:butt;stroke-linejoin:round;stroke:#000;stroke-opacity:1;stroke-miterlimit:10" d="M 84.912192 125.063225 L 95.638922 168.523784 L 52.628194 156.177663 Z M 84.912192 125.063225" transform="matrix(0.564449,0,0,0.564449,5.059717,2.40043)"/></g></svg>
```

Comment:
Missing an endline. This could cause some issues with some IDE's and configurations.
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="128pt" height="128pt" version="1.1" viewBox="0 0 128 128"><g id="surface1"><path style="fill:none;stroke-width:4;stroke-linecap:butt;stroke-linejoin:round;stroke:#000;stroke-opacity:1;stroke-miterlimit:10" d="M 63.022741 200.607088 L 19.804399 25.671417 L 193.037634 75.547253 Z M 63.022741 200.607088" transform="matrix(0.564449,0,0,0.564449,5.059717,2.40043)"/><path style="fill:none;stroke-width:4;stroke-linecap:butt;stroke-linejoin:round;stroke:#000;stroke-opacity:1;stroke-miterlimit:10" d="M 106.393334 50.612796 L 127.978284 138.108313 L 41.41703 113.160014 Z M 106.393334 50.612796" transform="matrix(0.564449,0,0,0.564449,5.059717,2.40043)"/><path style="fill:none;stroke-width:4;stroke-linecap:butt;stroke-linejoin:round;stroke:#000;stroke-opacity:1;stroke-miterlimit:10" d="M 84.912192 125.028622 L 74.185461 81.568063 L 117.19619 93.914184 Z M 84.912192 125.028622" transform="matrix(0.564449,0,0,0.564449,5.059717,2.40043)"/><path style="fill:none;stroke-width:4;stroke-linecap:butt;stroke-linejoin:round;stroke:#000;stroke-opacity:1;stroke-miterlimit:10" d="M 63.458731 38.155947 L 74.178541 81.616507 L 31.174733 69.270386 Z M 63.458731 38.155947" transform="matrix(0.564449,0,0,0.564449,5.059717,2.40043)"/><path style="fill:none;stroke-width:4;stroke-linecap:butt;stroke-linejoin:round;stroke:#000;stroke-opacity:1;stroke-miterlimit:10" d="M 149.473267 62.931234 L 160.193077 106.391793 L 117.189269 94.045673 Z M 149.473267 62.931234" transform="matrix(0.564449,0,0,0.564449,5.059717,2.40043)"/><path style="fill:none;stroke-width:4;stroke-linecap:butt;stroke-linejoin:round;stroke:#000;stroke-opacity:1;stroke-miterlimit:10" d="M 84.912192 125.063225 L 95.638922 168.523784 L 52.628194 156.177663 Z M 84.912192 125.063225" transform="matrix(0.564449,0,0,0.564449,5.059717,2.40043)"/></g></svg>

```